### DISCUSSION TITLE
Is it possible to use onStartReached callback?

### DISCUSSION THREAD

**Discussion by guihendias** [Shopify/flash-list#1193] [Q&A]
Is there a way to use `onStartReached` with Flash List? 

I'm asking it because we need to go to a message in a chat, let's say, on page 5, but can scroll up and down to fetch other pages, bidirectional.

Anyone has done that or knows how can we achieve it with Flash List?

TIA.

**Comment by hungtranauthentia** [comment]
This feature is much needed! 🙏
